pkg/controllers/termination/eviction.go

```diff
@@ -92,17 +92,18 @@ func (e *EvictionQueue) evict(ctx context.Context, nn types.NamespacedName) bool
 		ObjectMeta: metav1.ObjectMeta{Name: nn.Name, Namespace: nn.Namespace},
 	})
 	if errors.IsInternalError(err) { // 500
-		logging.FromContext(ctx).Errorf("Could not evict pod %s due to PDB misconfiguration error.", nn.String())
+		logging.FromContext(ctx).Errorf("Could not evict pod %s due to PDB misconfiguration error", nn)
 		return false
 	}
 	if errors.IsTooManyRequests(err) { // 429
-		logging.FromContext(ctx).Debugf("Did not evict pod %s due to PDB violation.", nn.String())
+		logging.FromContext(ctx).Errorf("Did not evict pod %s due to PDB violation", nn)
```

Comment:
I don't think we should be considering this an `Errorf` since it's an expected function of PDBs. Since it's fairly noisy, I'd like to keep this at the `Debug` level.

Reply:
Reverted this back to a 429 in the most recent push.